xml/containers-docker-registry.xml

```diff
@@ -92,57 +92,52 @@
   </variablelist>
  </sect1>
  <sect1 xml:id="sec-docker-registry-installation">
-  <title>Installing and Setting Up &dreg;</title>
+  <title>Running a &dreg;</title>
 
-  <procedure>
-   <step>
-    <para>
-     Install the <systemitem>docker-distribution-registry</systemitem> package.
-     This package is in &ph;. To enable &ph;, run the following commands:
-    </para>
-<screen>&prompt.sudo;SUSEConnect --product PackageHub/&productnumbershort;/x86_64
-&prompt.sudo;zypper refresh
-</screen>
-    <para>
-     Install <systemitem>docker-distribution-registry</systemitem>:
-    </para>
-<screen>&prompt.sudo;zypper install docker-distribution-registry</screen>
-   </step>
-   <step>
-    <para>
-     To automatically start the &dreg; at boot time:
-    </para>
-<screen>&prompt.sudo;systemctl enable registry</screen>
-   </step>
-   <step>
-    <para>
-     Start the &dreg;:
-    </para>
-<screen>&prompt.sudo;systemctl start registry</screen>
-   </step>
-  </procedure>
+  <para>
+   The &susereg; provides a container image that makes it possible to run a
+   local &dreg; as a container. Before you start a container, create a
+   <filename>config.yml</filename> file with the following example
+   configuration:
+  </para>
+
+<screen>version: 0.1
+log:
+  level: info
+storage:
+  filesystem:
+    rootdirectory: /var/lib/docker-registry
+http:
+  addr: 0.0.0.0:5000</screen>
 
   <para>
-   The &dreg; configuration is defined in the
-   <filename>/etc/registry/config.yml</filename> file.
+   Also create an empty directory necessary to map the
+   <filename>/var/lib/docker-registry</filename> directory outside the
+   container. This directory is used for storing container images.
   </para>
 
   <para>
-   With the default configuration, the registry listens on ports
-   <literal>5000</literal> and stores the container images in
-   <filename>/var/lib/docker-registry</filename>.
+   Run the following command to pull the registry container image from the
+   &susereg; and start a container that can be accessed on port 5000:
   </para>
 
-  <note>
-   <title>Incompatible Versions of the &deng; and &dreg;</title>
-   <para>
-    &dreg; 2.3 is not compatible with the &deng; versions older than 1.10,
-    because v2 manifests were only introduced with the &deng; 1.10. As the
-    &deng; and the &dreg; can be installed on different machines, the versions
-    might be incompatible. If you experience communication errors between the
-    &deng; and the &dreg;, update both to the latest versions.
-   </para>
-  </note>
+<screen>podman run -d --restart=always --name registry -p 5000:5000  -v /<replaceable>PATH</replaceable>/config.yml:/etc/docker/registry/config.yml -v /<replaceable>PATH</replaceable>/<replaceable>DIR</replaceable>:/var/lib/docker-registry registry.suse.com/sles12/registry:2.6.2</screen>
```

Comment:
this will search in the default registry which is registry.opensuse.org, add the registry URL here as well. See  bsc#1178770 for a bug to change that. Using the registry.suse.com URL is safe bet here.

Reply:
Ah, of course! My bad.

Reply:
Wait, no, I don't quite understand that. The `registry.suse.com/sles12/registry:2.6.2` is explicitly specified in the command, and it actually works. Am I missing something here?

Reply:
Indeed, sorry - missed that in the overlong line. I looked earlier.

Reply:
Yup, I need to chop the command to make it easier to read.